<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>491</width>
    <height>377</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton_sendWav">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>240</y>
      <width>161</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar arquivo .wav</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_piano">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>471</width>
      <height>221</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QSlider" name="verticalSlider_buzzerVolume">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>40</y>
       <width>20</width>
       <height>161</height>
      </rect>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
     <property name="value">
      <number>32767</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QProgressBar" name="progressBar_buzzerVolume">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>21</width>
       <height>161</height>
      </rect>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
     <property name="value">
      <number>32767</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_C4">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>C4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_D4">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>D4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_E4">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>E4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_F4">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>F4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_G4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>G4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_A4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>A4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_B4">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>B4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_C5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>C5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_B3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>41</width>
       <height>201</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px;}</string>
     </property>
     <property name="text">
      <string>B3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Cs4">
     <property name="geometry">
      <rect>
       <x>72</x>
       <y>10</y>
       <width>37</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px}</string>
     </property>
     <property name="text">
      <string>C#4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Ds4">
     <property name="geometry">
      <rect>
       <x>112</x>
       <y>10</y>
       <width>37</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px}</string>
     </property>
     <property name="text">
      <string>D#4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Fs4">
     <property name="geometry">
      <rect>
       <x>192</x>
       <y>10</y>
       <width>37</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px}</string>
     </property>
     <property name="text">
      <string>F#4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_Gs4">
     <property name="geometry">
      <rect>
       <x>232</x>
       <y>10</y>
       <width>37</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px}</string>
     </property>
     <property name="text">
      <string>G#4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_As4">
     <property name="geometry">
      <rect>
       <x>272</x>
       <y>10</y>
       <width>37</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {text-align: bottom; padding-bottom: 12px}</string>
     </property>
     <property name="text">
      <string>A#4</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_volume">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Volume</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="Line" name="line_pianoDivision">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>10</y>
       <width>16</width>
       <height>201</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_playWav">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Tocar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_pauseWav">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>270</y>
      <width>81</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_nameWav">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>240</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nome:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_durationWav">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Duração:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_sizeWav">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>291</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tamanho:</string>
    </property>
   </widget>
   <widget class="Line" name="line_controlsDiv">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>471</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_connect">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>91</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Conectar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_statusIsConnect">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>325</y>
      <width>361</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Status: Desconectado</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
